soba/feb/2023-02-26-모음제거.java

```diff
@@ -0,0 +1,10 @@
+package feb;
+
+class 모음제거 {
+    public String solution(String my_string) {
+        String answer = my_string;
+        answer = answer.replaceAll("[aeiou]","");
+        
+        return answer;
```

Comment:
answer에 할당안하고 바로 리턴하면 어떤가여?

```suggestion
        return answer.replaceAll("[aeiou]","");
```